### PR TITLE
feat : added alerts

### PR DESCRIPTION
**Pull request by iamalhera**

### CODE REVIEW

File: src/components/alert/alert.html [hunk 1]
```diff
@@ -4,10 +4,69 @@
...
+    </div>
+    <div class="container-column">
+        <div class="alert primary-alert"><i class="fas fa-solid fa-circle-info"></i>Primary Alert</div>
+        <div class="alert successfull-alert"><i class="fas fa-solid fa-circle-check"></i> Successfull Alert</div>
+        <div class="alert warning-alert"><i class="fas fa-solid fa-triangle-exclamation"></i> Warning Alert</div>
+        <div class="alert danger-alert"><i class="fas fa-solid fa-skull-crossbones"></i> Danger Alert</div>
+    </div>
+</section>
+</div>
+<i class="fa-light fa-circle-info"></i>
```
Comment (nikhiljugale007) [inline src/components/alert/alert.html]: what is this icon doing?
I am not able to see this icon in UI.

**Review by nikhiljugale007 — COMMENTED** [review]
UI and Code is good.
One suggestion added in code.
One UI suggestion that you can use different color for warning and danger alert  so that they can be easily identified.

File: src/components/alert/alert.html [hunk 2]
```diff
@@ -4,10 +4,69 @@
...
+<section class="background-white">
+    <h3>Exampes of Alert</h3>
+    <div class="section-description ">
+        Alerts are available in four types i.e Primary Alert, Successful Alert, Danger Alert and Warning Alert. To use this component, you have to add <span class="highlighted">alert</span>, according to your need you can use these classes <span class="highlighted">primary-alert</span>, <span class="highlighted">successfull-alert</span>, <span class="highlighted">danger-alert</span>, <span class="highlighted">warning-alert</span>.
+    </div>
+    <div class="container-column">
+        <div class="alert primary-alert"><i class="fas fa-solid fa-circle-info"></i>Primary Alert</div>
+        <div class="alert successfull-alert"><i class="fas fa-solid fa-circle-check"></i> Successfull Alert</div>
+        <div class="alert warning-alert"><i class="fas fa-solid fa-triangle-exclamation"></i> Warning Alert</div>
+        <div class="alert danger-alert"><i class="fas fa-solid fa-skull-crossbones"></i> Danger Alert</div>
```
Comment (yaswanthmyneni) [inline src/components/alert/alert.html]: give some space between icon and text. other than that remaining is good.

File: src/components/alert/alert.html [hunk 3]
```diff
@@ -4,10 +4,69 @@
...
+    </div>
+    </div>
+
+<section class="background-white">
+    <h3>Exampes of Alert</h3>
+    <div class="section-description ">
+        Alerts are available in four types i.e Primary Alert, Successful Alert, Danger Alert and Warning Alert. To use this component, you have to add <span class="highlighted">alert</span>, according to your need you can use these classes <span class="highlighted">primary-alert</span>, <span class="highlighted">successfull-alert</span>, <span class="highlighted">danger-alert</span>, <span class="highlighted">warning-alert</span>.
+    </div>
+    <div class="container-column">
+        <div class="alert primary-alert"><i class="fas fa-solid fa-circle-info"></i>Primary Alert</div>
```
Comment (veereshbv04) [inline src/components/alert/alert.html]: it is info- circle not fa-circle-info

File: src/components/alert/alert.html [hunk 4]
```diff
@@ -4,10 +4,69 @@
...
+        <div class="alert successfull-alert"><i class="fas fa-solid fa-circle-check"></i> Successfull Alert</div>
```
Comment (veereshbv04) [inline src/components/alert/alert.html]: use fas fa-solid fa-check-circle

File: src/components/alert/alert.html [hunk 5]
```diff
@@ -4,10 +4,69 @@
...
+        <div class="alert warning-alert"><i class="fas fa-solid fa-triangle-exclamation"></i> Warning Alert</div>
```
Comment (veereshbv04) [inline src/components/alert/alert.html]: use fas fa-solid fa-exclamation-triangle

File: src/components/alert/alert.html (hunk 1 above)
Comment (iamalhera) [reply src/components/alert/alert.html]: deleted that icon

File: src/components/alert/alert.html (hunk 3 above)
Comment (iamalhera) [reply src/components/alert/alert.html]: changed it

File: src/components/alert/alert.html (hunk 5 above)
Comment (iamalhera) [reply src/components/alert/alert.html]: did that

File: src/components/alert/alert.html (hunk 4 above)
Comment (iamalhera) [reply src/components/alert/alert.html]: did that

File: src/components/alert/alert.html (hunk 2 above)
Comment (iamalhera) [reply src/components/alert/alert.html]: okay thanks for pointing out

**Pull request merged**